bundle/manifests/mcg-osd-deployer.clusterserviceversion.yaml

```diff
@@ -14,10 +14,10 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
```

Comment:
I've reverted it because in my case the version I'm using is based on the latest commit, which is why it says unknown.

![image](https://user-images.githubusercontent.com/33557095/159262722-f939225e-e041-4866-8dc9-246f68baff89.png)